src/HelsinkiProfiiliUserData.php

```diff
@@ -425,7 +471,7 @@ public function getUserProfileData(bool $refetch = FALSE): ?array {
    *   String array containing token parameters.
    */
   private function getProfileTokenParams(): array {
-    $appEnv = ApplicationHandler::getAppEnv();
+    $appEnv = self::getAppEnv();
```

Comment:
AFAIK, this could be refactored to use [Environment resolver service](https://github.com/City-of-Helsinki/drupal-module-helfi-api-base/blob/main/documentation/environment-resolver.md) (which is already injected to this service).

Something like:

```php
private function getProfileTokenParams(): array {
  $appEnv = $this->environmentResolver->getActiveEnvironmentName();

  $endpointAudience = 'profile-api-test';

  if ($appEnv === \Drupal\helfi_api_base\Environment\EnvironmentEnum::Prod->value) {
    $endpointAudience = 'profile-api';
  }
  if ($appEnv === \Drupal\helfi_api_base\Environment\EnvironmentEnum::Stage->value) {
    $endpointAudience = 'profile-api-stage';
  }

  return [
    'audience' => $endpointAudience,
    'grant_type' => 'urn:ietf:params:oauth:grant-type:uma-ticket',
    'permission' => '#access',
  ];
}
```